AdminUi/.gitignore

```diff
@@ -0,0 +1,9 @@
+pubspec_overrides.yaml
+.dart_tool
+build
+.flutter-plugins
+.flutter-plugins-dependencies
+.DS_Store
+
+# config.json for connecting to the admin api
+config.json
```

Comment:
@jkoenig134 Shouldn't we commit this with the connection to the local server?

Reply:
With "this" I mean the `config.json` file

Reply:
Sounds overcomplicated. Why not just committing the config.json file?
But if that's the way to go in flutter, I'm okay with it.

Reply:
As mac users will have a different config I always have to make sure to not commit it.